src/components/Dashboard/TodoList.vue

```diff
@@ -1518,7 +1561,10 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin) {
         payErrorObj,
         isPayCompleted: (paymentStatusCode === 'COMPLETED'),
         isEmptyFiling: !haveData,
-        nameRequest: this.getNameRequest
+        nameRequest: this.getNameRequest,
+        submitter: header.submitter,
+        submittedDate: new Date(header.date), // API format
+        latestReviewComment: header?.latestReviewComment
```

Comment:
New properties needed in case changes are needed, ie:

![image](https://github.com/user-attachments/assets/cad1ad23-fc39-469e-92a0-0068ef697fa1)